pkg/functionconfig/types.go

```diff
@@ -416,6 +415,9 @@ type Spec struct {
 	// InitContainers are specialized containers that run before app containers in a Pod
 	// Init containers can contain utilities or setup scripts not present in an app image
 	InitContainers []*v1.Container `json:"initContainers,omitempty"`
+
+	// Deprecated - remove in 1.13.x
+	Avatar string `json:"avatar,omitempty"`
```

Comment:
on create/update function
if avatar has content, warn

Reply:
Doesn't `omitempty` do exactly that already?

Reply:
oh, sorry, indeed it does